### PR TITLE
Docs 6 16

### PR DESCRIPTION
**Pull request by taylorsperry**
Update for "Cody-only" app version, with documentation for rate limiting and known issues. 

## Test plan

n/a docs update

### CODE REVIEW

**Review by chwarwick — APPROVED** [review]
LGTM

File: doc/app/index.md [hunk 1]
```diff
@@ -34,44 +34,54 @@ The app is a free, lightweight, native desktop version of Sourcegraph that conne
...
-* [Create a GitHub issue](https://github.com/sourcegraph/app/issues/new)
+There are several forms of rate limiting that help us control costs for free versions of Cody. We expect to relax these limits as we continue development on [Cody Gateway](../cody/explanations/cody_gateway.md).
 
-## Upgrading
+### Cody Chat
+Interactions with Cody Chat (whether in the app UI or in the editor extension) are capped at 100 requests per day. 
 
-We're shipping new releases of the app quickly, and you should get prompts to upgrade automatically. Let us know if you have any issues!
+### Completions
+Cody completions (autocomplete powered by Cody) are capped at 100 requests per day. Learn more about [Cody Completions](../cody/completions.md).
```
[the remark this answers is not in the document]
Comment (chwarwick) [reply doc/app/index.md:68]: @taylorsperry  @philipp-spiess  The limit is the same as dotcom and if it is adjusted for a specific user on dotcom usage though app will also be increased/decreased.

[the remark this answers is not in the document]
Comment (chwarwick) [reply doc/app/index.md:68]: Sorry I missed that first time though, I read completions and thought chat.

[the remark this answers is not in the document]
Comment (philipp-spiess) [reply doc/app/index.md:68]: Ah makes sense. Yeah we have two separate limits for autocompletions and chat but on the backend we call everything completions so it's super confusing. https://about.sourcegraph.com/blog/increasing-the-completions-rate-limit

**Pull request merged**